//package core.io.protocol

message DceStageExplore
{
	optional int32 type = 1;//1请求数据 2收获 3领取掉落
}
